Lib/test/test_subprocess.py

```diff
@@ -58,13 +58,16 @@ def setUp(self):
         # doesn't crash on some buildbots (Alphas in particular).
         support.reap_children()
 
-    def tearDown(self):
-        for inst in subprocess._active:
-            inst.wait()
-        subprocess._cleanup()
-        self.assertFalse(subprocess._active, "subprocess._active not empty")
-        self.doCleanups()
-        support.reap_children()
+    if not mswindows:
+        # No need to do anything for Windows. See a note for
+        # subprocess._cleanup() for more info.
```

Comment:
Please only skip the code which rely on _active: calling support.reap_children()  on Windows is still useful.